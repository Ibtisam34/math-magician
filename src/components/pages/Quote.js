import React from 'react';

export default function Quote() {
  return (
    <div className="quote--pages-container">
      <div>
        Mathematics is not about numbers, equations, computations,
        or algorithms: it is about understanding. â€“William Paul Thurston
      </div>
    </div>
  );
}
