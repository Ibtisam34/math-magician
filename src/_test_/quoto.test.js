import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/pages/Quote';

it('renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
