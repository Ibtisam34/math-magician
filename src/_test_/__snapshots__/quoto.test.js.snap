// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote--pages-container"
>
  <div>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </div>
</div>
`;
